dltxio/blockchain-id
# ID-6

To test 
1, run expo install  and start app 
2. navigate to claims choose one of the claim type from the list and  press the buttong to add document from device. expect open new window to choose the file from localstorage.
